microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -0,0 +1,193 @@
+
+Now in Microsoft Edge 114, you no longer need to select an individual **Recalculate Style** event to see the selectors that were recalculated during that event. Rather, the **Selector Stats** tab in the bottom pane of the **Performance** tool automatically aggregates the data across all of the **Recalculate Style** events in the recorded profile. As you zoom into specific parts of the profile, the **Selector Stats** tab updates to only show data from the portion of the profile that you are currently analyzing. 
+
+Additionally, a new column called **Style sheet** has been added to the **Selector Stats** tab and contains links for each selector back to the stylesheet where the selectors are defined. 
+
+![Selector Stats are now aggregated across Recalculate Style events within the currently displayed section of the recorded profile](./devtools-114-images/aggregate-selector-stats.png)
+
+<!-- todo
+1. Open Devtools and navigate to the performance tool
+2. Hit the capture settings button
+3. Check the box at the top that says "Enable advanced rendering instrumentation (slow)"
+4. Click on a purple "Recalculate style" box
+5. Click on the Selector stats button on the bar below
+-->
+
+Thank you for using the **Selector Stats** feature and sharing your feedback with us in [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)!
+
+See also:
+* [Analyze selector performance during Recalculate Style events](../../../evaluate-performance/selector-stats.md)
+* [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)
+
+<!-- ====================================================================== -->
+## The Issues tool warns you when CSS properties trigger reflow 
+
+<!-- Subtitle: Reflow is when the browser re-calculates the positions and geometries of elements in the page and is user-blocking so limit reflow as much as possible for better performance.-->
+
+<!-- Reviewer: Vidal Guillermo Diazleal Ortega -->
+
+Reflow is the name of the web browser process for re-calculating the positions and geometries of elements in the document, for the purpose of re-rendering part or all of the document. Because reflow is a user-blocking operation in the browser, from a performance perspective, we recommend limiting reflow as much as possible to keep your web content smooth and interactable. Some CSS properties don't trigger reflow because they run on the compositor thread in the browser, like [transform](https://developer.mozilla.org/docs/Web/CSS/transform) and [opacity](https://developer.mozilla.org/docs/Web/CSS/opacity).
```

Comment:
`Some CSS properties don't trigger reflow because they run on the compositor thread in the browser, like transform and opacity`

Should be

`Some CSS properties don't trigger Layout because they run on the compositor thread in the browser, like transform and opacity, but they can trigger Paint which, when used in combination with CSS animations, can lead to a negative performance impact`

Reply:
I will apply this comment to [PR 2618](https://github.com/MicrosoftDocs/edge-developer/pull/2618)

Reply:
2
@jenna-sasson - this comment thread can be closed.

gh rendered section in other PR's branch:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/whats-new-114/microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md#the-issues-tool-and-styles-pane-warn-about-css-properties-that-trigger-layout